### PR TITLE
[Improvement] wordpress: CloudFront prefix list to control traffic

### PR DESCRIPTION
**Pull request by andreaswittig**
AWS announced prefix lists containing the IP address ranges of CloudFront. Using this to control traffic from CloudFront to the ALB.

### CODE REVIEW

File: wordpress/wordpress-ha-aurora.yaml [hunk 1]
```diff
@@ -256,6 +271,8 @@ Conditions:
   HasManagedPolicyArns: !Not [!Equals [!Ref ManagedPolicyArns, '']]
   HasEFSBackupRetentionPeriod: !Not [!Equals [!Ref EFSBackupRetentionPeriod, 0]]
   HasWAF: !Not [!Equals [!Ref ParentWAFStack, '']]
+  HasCloudFrontPrefixList: !Not [!Or [!Equals [!Ref AWS::Region, 'ap-northeast-3'], !Equals [!Ref AWS::Region, 'ap-southheast-3']]]
```
Comment (michaelwittig) [inline wordpress/wordpress-ha-aurora.yaml:274]: does this work?

  HasCloudFrontPrefixList: !Not [!Condition HasNotCloudFrontPrefixList]
   HasNotCloudFrontPrefixList: !Equals [!FindInMap[], '']

Comment (andreaswittig) [reply wordpress/wordpress-ha-aurora.yaml]: Yes, great idea!

File: wordpress/wordpress-ha.yaml [hunk 2]
```diff
@@ -407,6 +422,8 @@ Conditions:
   HasEFSBackupRetentionPeriod: !Not [!Equals [!Ref EFSBackupRetentionPeriod, 0]]
   HasRDSPerformanceInsights: !Equals [!FindInMap [DBServerInstanceTypeMap, !Ref DBServerInstanceType, PerformanceInsights], true]
   HasWAF: !Not [!Equals [!Ref ParentWAFStack, '']]
+  HasCloudFrontPrefixList: !Not [!Equals [!FindInMap ['RegionMap', !Ref 'AWS::Region', 'PrefixListCloudFront'], '']]
+  HasNotCloudFrontPrefixList: !Equals [!FindInMap ['RegionMap', !Ref 'AWS::Region', 'PrefixListCloudFront'], '']
```
Comment (michaelwittig) [inline wordpress/wordpress-ha.yaml:426]: Please update as well

**Review by michaelwittig — CHANGES_REQUESTED** [review]

File: wordpress/wordpress-ha.yaml (hunk 2 above)
Comment (andreaswittig) [reply wordpress/wordpress-ha.yaml]: ✅

**Review by michaelwittig — APPROVED** [review]

**Pull request merged**